File: ios/Classes/ErrorCode.swift
//
//  ErrorCode.swift
//  integration_test
//
//  Created by ipification on 1/20/21.
//

import Foundation

/// Error codes exposed to Flutter when native IPification operations fail.
enum ErrorCode  :String {
    case COVERAGE_UNAVAILABLE = "coverage_unavailable"
    case COVERAGE_ERROR = "coverage_error"
    case AUTHENTICATE_FAIL = "authenticate_fail"
    case AUTHENTICATE_IM_CANCEL = "im_canceled"
}

File: android/src/main/kotlin/com/ipification/plugin/ErrorCode.kt
package com.ipification.plugin

/**
 * Error codes exposed to Flutter when native IPification operations fail.
 *
 * @property code String value used as the FlutterError code.
 */
enum class ErrorCode (val code:String){
    COVERAGE_UNAVAILABLE("check_coverage_unavailable"),
    COVERAGE_ERROR("check_coverage_error"),
    AUTHENTICATE_FAIL("authentication_failed"),
    AUTHENTICATE_IM_CANCEL("im_canceled")
}
